### ISSUE TITLE
Add template option for Create Note command

### ISSUE THREAD

**Issue by Perchycs** [marcjulianschwarz/obsidian-raycast#88]
I've been trying to make this work for a while, and as far as I can tell there isn't a way to achieve what I want easily.

### Current Workflow
I have been using the extension to quickly create a 'fleeting note' which I can quickly fill the content of using the form for the create new note command. This works great!

### Issue
I am wanting to add a 'created_date' field as frontmatter, because dataview's 'file.cdate' doesn't work well when syncing notes across multiple devices. The 'cdate' reflects the file's creation date on each device (ie. when it was synced to that device), which can differ from the actual note's creation date.

### Current Limitations
**Using the 'default note content' option:**
- If 'fill form with defaults' is enabled, the default frontmatter appears in the content box, which ends up looking cluttered.
- If 'fill form with defaults' is disabled, the default value is overwritten when adding content.
- Adding tags via the form results in the default frontmatter being added after the tags, causing formatting issues (example [here](https://imgur.com/UkSpl9w)).

**Using the feature introduced from #14** 
- Using #14 is not ideal, as the whole reason I have been loving this extension is that you can do everything on the form without switching into Obsidian.

### Feature Requests
**New Setting:** Add a 'text to append to the beginning of the note' option, allowing users to input frontmatter or initial text (e.g., a templater command). This could function like the similarly named option for the 'append task' command.
**Tag Handling:** Ensure that any tags selected in the form are integrated into this frontmatter, avoiding multiple frontmatter blocks. This might reduce the relevance of the 'default tags' setting, as users can add a 'tags: [your_tag]' field in the appended text.

---

Thanks so much, really appreciate the work you have done to create & improve this extension, it is such a useful addition to Obsidian!

**Comment by KevinBatdorf** [comment]
> If 'fill form with defaults' is disabled, the default value is overwritten when adding content.

Do you think this is a bug? Where the content should show be there? Seems like it could be to me.

**Comment by Perchycs** [comment]
Hard to say, I feel like it was intended like that, as it is the 'default' content, so if you create a note with nothing in the content field, then the default content will be selected. 

Makes sense to have this for a default note title, though I am not sure it necessarily makes as much sense with the note content, as I can't see any use for creating a note with the same default content.

**Comment by marcjulianschwarz** [comment]
@Perchycs yes, it was intended. The idea behind having a default content was to have dynamic placeholders in there (e.g. clipboard content or selected text).
But you are right, it would make a lot more sense to not have a "default content" but more of a "template".

Your use case could look like this:

```
---
created-date: {date}
tags:
    - test
    - test2
---

{content}
```

One problem I see here is that you can't really edit these big templates well in the Raycast settings. For example there is no way to easily enter new lines (except maybe by writing `/n` every time. 

Will check how something like that could work out.

**Comment by Perchycs** [comment]
That sounds great!

Is there a way you could pull the template from an obsidian note, rather than writing it in the Raycast settings?
